src/app/globals.css

```
@@ -66,15 +80,31 @@
+* {
+  box-sizing: border-box;
 }
 
 body {
-  background: var(--background);
+  margin: 0;
+  min-height: 100vh;
   color: var(--foreground);
   font-family: var(--font-sans);
+
+  background-color: var(--pf-bg-base);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)